unstructured/documents/elements.py

```diff
@@ -553,4 +561,12 @@ class Footer(Text):
     "Table": Table,
     "Header": Header,
     "Footer": Footer,
+    "Caption": Text,
+    "Footnote": Footer,
+    "Formula": Formula,
+    "List-item": ListItem,
+    "Page-footer": Footer,
+    "Page-header": Header,  # Title?
```

Comment:
good question in the comment. have any images available where yolox identified a `Page-header`?

Reply:
Actually no, I only found a lot of section-headers but any page-header 🤔

Reply:
kk